transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -87,7 +87,7 @@ extern int epoll_create1(int flags) __attribute__((weak));
 extern int epoll_pwait2(int epfd, struct epoll_event *events, int maxevents, const struct timespec *timeout, const sigset_t *sigmask) __attribute__((weak));
 
 #ifndef __USE_GNU
-struct mmsghdr {
+struct _mmsghdr {
```

Comment:
Where is this leading-underscore variant defined? I'm not finding any docs on it. I only see [sendmmsg](https://www.mankier.com/2/sendmmsg) defining the non-leading-underscore symbol.